packages/components/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@kaizen/components",
+  "version": "1.0.0",
```

Comment:
Hmm...since this isn't really housing anything, should we make it `0.0.0`? 🤔 Your choice though.

Reply:
Hm don't think anyone can install a `0.0.0` so it'd have to be `0.0.1`, but if that's the case the follow up question would be what would make it `1.0.0`? The first component? Despite it not breaking anything.

Reply:
I think `0` to `1` is a special case, is it not? It's essentially saying it's in Beta.
Although one caveat is that any breaking changes made while in `v0` is that it does not get bumped to `v1`. That must be done manually.

Up to you :)